operator/cmd/mesh/testdata/manifest-generate/data-snapshot/profiles/preview.yaml

```diff
@@ -8,6 +8,6 @@ spec:
     telemetry:
       v2:
         metadataExchange:
-          wasmEnabled: false
+          wasmEnabled: true
```

Comment:
Is this change related to the binding issue?

Reply:
No, its related to how the istioctl tests work. See the file name, `data-snapshot` - to avoid having changes to the testdata every time the charts change, we keep a copy of them so its periodically updated. I had to update this copy when making this change or all the tests fail since the old copy is now invalid

Reply:
Thanks!